core/action.class.inc.php

```diff
@@ -168,6 +170,54 @@ protected function DoCheckIfHasTrigger()
 			$this->m_aCheckWarnings[] = Dict::S('Action:WarningNoTriggerLinked');
 		}
 	}
+
+	/**
+	 * @since 3.2.0 N°5472 method creation
+	 */
+	public function DisplayBareRelations(WebPage $oPage, $bEditMode = false)
+	{
+		parent::DisplayBareRelations($oPage, false);
+
+		if ($oPage instanceof iTopWebPage) {
+			$this->GenerateLastExecutionsTab($oPage, $bEditMode);
+		}
+	}
+
+	/**
+	 * @since 3.2.0 N°5472 method creation
+	 */
+	protected function GenerateLastExecutionsTab(iTopWebPage $oPage, $bEditMode)
+	{
+		$oRouter = Router::GetInstance();
+		$sActionLastExecutionsPageUrl = $oRouter->GenerateUrl('notifications.action.last_executions_tab', ['actionid' => $this->GetKey()]);
```

Comment:
```suggestion
		$sActionLastExecutionsPageUrl = $oRouter->GenerateUrl('notifications.action.last_executions_tab', ['action_id' => $this->GetKey()]);
```

The convention is to put an underscore.

Reply:
Done in 4862977d